### ISSUE TITLE
Support for native advertising

### ISSUE THREAD

**Issue by omriariav** [WICG/privacy-preserving-ads#69]
Hi all,

I oversee user data as the product manager at @taboola, leading our preparations for a cookieless world. Over the past year, we have actively collaborated with the Privacy Sandbox teams (Chrome and Android) to advocate for the full support of native advertising.

Native ads blend into the website's content, matching the platform's look, feel, and function. These ads appear part of the user’s natural browsing experience, making them less intrusive. We use our [JS library](https://developers.taboola.com/web-integrations/docs/welcome), which publishers integrate into the page. We render mid-article widgets and the below article's endless feed ([example](https://feed.taboola.com/feed-yours)). In Taboola, next to ads, we also recommend organic items of the publisher, aiming to boost engagement metrics. The blend between ads differs per impression according to our content recommendation decisions. 

Taboola supports various campaigns, including different forms of retargeting. We provide our own JavaScript library (pixel) that advertisers can place on their landing pages to enable these features.

We have the highest respect for the work done so far, but we think it didn't consider native advertising, focusing only on display advertising needs.

We would like to highlight the following **main** gaps in regards to retargeting and audiences:

- Native look and feel - The Fenced Frame solution must allow the `renderURL` to use publisher page look and feel styles ([https://github.com/WICG/turtledove/issues/741](https://github.com/WICG/turtledove/issues/741)). 
Ad coordination—In native advertising, the number of ads per placement is higher than in display ads, potentially creating endless opportunities for auctions. Being aware of the status of current/previous auctions is critical for de-duplication, demand diversity, and ad quality rules. We need to ensure the server-side auction design takes this into consideration 
  - https://github.com/WICG/turtledove/issues/1199
  - https://github.com/WICG/turtledove/issues/1097
  - https://github.com/WICG/turtledove/issues/1074
  - https://github.com/WICG/turtledove/issues/792#issuecomment-2340425977 
- Suppression support - in native advertising, the order of ads is based on ranking. When we serve suppression (exclusion retargeting) and the criteria are not met, we must maintain the order of ads. We will need to ensure the server-side auction can support the native use case
  - https://github.com/WICG/turtledove/issues/896#issuecomment-2272778495 
  - https://github.com/WICG/turtledove/issues/1096

We will be happy to collaborate and engage to promote these needs.
Thanks!

**Comment by ptetali2** [comment]
Hi, thanks for bringing this up- we plan on discussing this at our next public call on Thurs 10/17 at 8 AM PST. See more details [here](https://github.com/WICG/privacy-preserving-ads/issues/50).

**Comment by omriariav** [comment]
Thurs 10/17 is a Jewish holiday and the team here will be away. Can we discuss this in the 10/30 meeting? 

Thanks

**Comment by ptetali2** [comment]
> Thurs 10/17 is a Jewish holiday and the team here will be away. Can we discuss this in the 10/30 meeting?
> 
> Thanks

Got it! We will be discussing this in the 10/30 meeting

**Comment by omriariav** [comment]
Confirming we're going to review this on the 31st, right?

Thank you

**Comment by ptetali2** [comment]
> Confirming we're going to review this on the 31st, right?
> 
> Thank you

Yup confirmed for 10/31. To see agenda + join info see [here](https://docs.google.com/document/d/1Yy3KFtnUmqMRUmpmgBNGJUGtjkqRhokrLlYr1vdnYjw/edit?tab=t.0#heading=h.5yicbq39k7qo)

**Comment by ptetali2** [comment]
Hi @omriariav would you be available 12/12 to continue the discussion on Native Advertising?

**Comment by omriariav** [comment]
@ptetali2 yes, but @talbizit will represent us.

**Comment by ptetali2** [comment]
Hi @omriariav, would you or @talbizit be able to attend the 1/23 call to continue ideation on this topic?

**Comment by ptetali2** [comment]
Hi @talbizit would you be able to attend the 2/7 call to continue discussion on this?

**Comment by talbizit** [comment]
Yes, we will attend today

**Comment by michaelkleber** [comment]
Apologies, due to a last-minute conflict, I cannot attend today.

**Comment by ptetali2** [comment]
Hi @talbizit are there additional items that need to be discussed for this proposal? If so could you make it to the [4/17](https://docs.google.com/document/d/1Yy3KFtnUmqMRUmpmgBNGJUGtjkqRhokrLlYr1vdnYjw/edit?tab=t.0#heading=h.3csgsxoxsktc) call?

**Comment by omriariav** [comment]
@ptetali2, most of us are on Passover break. Can we check in on the following meeting?
